taskmanager/features/task.feature

```diff
@@ -1,26 +1,39 @@
 # language: en
-  Feature: Task management
```

Comment:
formating is default from PHPStorm in my opinion for simplicity we should follow them (2 spaces for indent)